### PR TITLE
Fix LGTM errors in extensions/

### PR DESCRIPTION
**Pull request by rsimha-amp**
Fixes the following LGTM errors:

- [extensions/amp-ad/0.1/amp-ad-3p-impl.js#L128](https://lgtm.com/projects/g/ampproject/amphtml/snapshot/a4d5dedaa4349a630d64ac4ffa41ed08b90f7efe/files/extensions/amp-ad/0.1/amp-ad-3p-impl.js?sort=name&dir=ASC&mode=heatmap&excluded=false#L128)
- [extensions/amp-analytics/0.1/requests.js#L166](https://lgtm.com/projects/g/ampproject/amphtml/snapshot/a4d5dedaa4349a630d64ac4ffa41ed08b90f7efe/files/extensions/amp-analytics/0.1/requests.js?sort=name&dir=ASC&mode=heatmap&excluded=false#L166)
- [extensions/amp-story/0.1/logging.js](https://lgtm.com/projects/g/ampproject/amphtml/snapshot/a4d5dedaa4349a630d64ac4ffa41ed08b90f7efe/files/extensions/amp-story/0.1/logging.js?sort=name&dir=ASC&mode=heatmap&excluded=false#L99)

Partial fix for #12597

### CODE REVIEW

File: extensions/amp-analytics/0.1/requests.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ export class RequestHandler {
       } else {
         preUrl = appendEncodedParamStringToUrl(preUrl, extraUrlParamsStr);
       }
-      return baseUrlTemplatePromise.then(preUrl => {
+      return baseUrlTemplatePromise.then(() => {
```
Comment (jridgewell) [inline extensions/amp-analytics/0.1/requests.js]: Bug! These are not equivalent!

Comment (rsimha-amp) [reply extensions/amp-analytics/0.1/requests.js]: That's right, I believe this is a bug in the original code. Adding @zhouyx, who introduced this in #11837:

See https://github.com/ampproject/amphtml/pull/11837/files#diff-b951eee683ce1cc97c5e22f9abd32f11R169

Also see https://lgtm.com/projects/g/ampproject/amphtml/snapshot/a4d5dedaa4349a630d64ac4ffa41ed08b90f7efe/files/extensions/amp-analytics/0.1/requests.js?sort=name&dir=ASC&mode=heatmap&excluded=false

**Comment by rsimha-amp** [comment]
Errors fixed: https://lgtm.com/projects/g/ampproject/amphtml/rev/pr-1b0d43fcefaf79aa59c71aa534f62766ac42343a

File: extensions/amp-analytics/0.1/requests.js (hunk 1 above)
Comment (zhouyx) [reply extensions/amp-analytics/0.1/requests.js]: Failed to remove unused in #11837. Had #12776 to fix that.

Comment (rsimha-amp) [reply extensions/amp-analytics/0.1/requests.js]: Thanks. File reverted.

Comment (rsimha-amp) [reply extensions/amp-analytics/0.1/requests.js]: @jridgewell @zhouyx PTAL

**Comment by rsimha-amp** [comment]
Errors fixed in https://lgtm.com/projects/g/ampproject/amphtml/rev/pr-050402326607b01a74faf3499b47412f5fb68cac

**Pull request merged**